tuist/tuist
# Add tuist cloud project create command

### Short description 📝

Implements a `tuist cloud project create` command.

See docs [here](https://github.com/tuist/docs/pull/15) to learn more about it.

The OpenAPI models have been generated using the following command:
```sh
mint run swift-openapi-generator generate Sources/TuistCloud/OpenAPI/cloud.yml --mode types --mode client --output-directory Sources/TuistCloud/OpenAPI
```

### How to test the changes locally 🧐

- Run `tuist cloud auth` if you're not already authenticated
- Run `tuist cloud project create some-project` to create a new project
- You can check at cloud.tuist.io the project has been created.

### Contributor checklist ✅

- [x] ~The change is tested via unit testing or acceptance testing, or both~ (There's not much to test, acceptance test would be best but currently, we don't have ones for cloud)

### Reviewer checklist ✅
